syft/frameworks/torch/mpc/aby3/aby3_helper.py

```diff
@@ -0,0 +1,65 @@
+import torch
+
+from syft.frameworks.torch.tensors.interpreters.replicated_shared import ReplicatedSharingTensor
+
+
+class ABY3Helper:
+    def bit_inject(rst_binary):
+        assert rst_binary.ring_size == 2
+
+        return_wrapper = rst_binary.is_wrapper
+        if return_wrapper:
+            rst_binary = rst_binary.child
+
+        players = rst_binary.players
+        shares_p1, shares_p2, shares_p3 = rst_binary.retrieve_pointers()
+        zeros = torch.zeros(shares_p1[0].shape, dtype=torch.long)
+
+        zeros_p1 = zeros.send(players[0])
```

Comment:
What else we can do here:
zero_share1 = gen_alpha3of3(players[0])
zero_share2 = gen_alpha3of3(players[1])
zero_share3 = gen_alpha3of3(players[2])
```
Where:
```zero_share1 + zero_share2 + zero_share3 = 0``` (using the corelated randomness)

And then we will have:
```
x1_shares = ((x_b_1, zero_share1), (zero_share2, zero_share3), (zero_share3, x_b_1))
x2_shares = ((zero_share1, x_b_2), (x_b_2, zero_share3), (zero_share3, zero_share1))
x3_shares = ((zero_share1, zero_share2), (zero_share2, x_b_3), (x_b_3, zero_share1))
```

The end result will be computed the same, only that we use the ```correlated randomness``` (also this require sending the zero_share1, zero_share2, zero_share3 to the other parties -- more communication overhead)